### ISSUE TITLE
FCI failing for CO in STO-3G

### ISSUE THREAD

**Issue by wpoely86** [psi4/psi4public#69]
I've used the following input:
```python
molecule CO
{
    C
    O 1 R
    units au
}

set {
    basis sto-3g
    guess read
}

Rrange= [ 0.6,  0.7,  0.8,  0.9,  1. ,  1.1,  1.2,  1.3,  1.4,  1.5,  1.6, 1.7,  1.8,  1.9,  2. ,  2.1,  2.2,  2.3,  2.4,  2.5,  2.6,  2.7, 2.8,  2.9,  3. ,  3.1,  3.2,  3.3,  3.4,  3.5,  3.6,  3.7,  3.8, 3.9,  4. ,  4.1,  4.2,  4.3,  4.4,  4.5,  4.6,  4.7,  4.8,  4.9, 5. ,  5.1,  5.2,  5.3,  5.4,  5.5,  5.6,  5.7,  5.8,  5.9,  6. , 6.1,  6.2,  6.3,  6.4,  6.5,  6.6,  6.7,  6.8,  6.9,  7. ,  7.1, 7.2,  7.3,  7.4,  7.5,  7.6,  7.7,  7.8,  7.9,  8. ,  8.1,  8.2, 8.3,  8.4,  8.5,  8.6,  8.7,  8.8,  8.9,  9. ,  9.1,  9.2,  9.3, 9.4,  9.5,  9.6,  9.7,  9.8,  9.9, 10. ]

table=Table(rows=["R"], cols=["E(SCF)", "E(FCI)"])

for R in Rrange:
    CO.R = R

    energy('fci')
    escf = get_variable('SCF TOTAL ENERGY')
    efci = get_variable('CI TOTAL ENERGY')

    table[R] = [escf, efci]

print table
```
The results looks like:
![Results plot](http://i.imgur.com/iupslMD.png)

Full CI shouldn't fail in this case? This was done with f7e55f09f7

**Comment by wpoely86** [comment]
OK, switching to `DIAG_METHOD RSP` solves it. 

The problem remains that PSI picks a very bad starting vector for it's Davidson solver.

**Comment by ryanmrichard** [comment]
Merged with issue #175
